parsel/selector.py

```diff
@@ -164,9 +223,18 @@ def extract(self):
                 return six.text_type(self.root)
 
     def register_namespace(self, prefix, uri):
+        """
+        Register the given namespace to be used in this :class:`Selector`.
+        Without registering namespaces you can't select or extract data from
+        non-standard namespaces. See examples below.
+        """
         self.namespaces[prefix] = uri
 
     def remove_namespaces(self):
+        """
+        Remove all namespaces, allowing to traverse the document using
+        namespace-less xpaths. See example below.
```

Comment:
"See example below" doesn't make much sense in a docstring. What about See :ref:\`Removing namespaces <...>\`?